src/test/java/com/ericsson/bss/cassandra/ecaudit/integration/ITVerifyAudit.java

```diff
@@ -531,6 +525,45 @@ public void testFailedStatementsAreLogged()
                         .containsAll(expectedAttemptsAndFails(statements));
     }
 
+    @Test
```

Comment:
I might be missing it, but I'd like to see a test case for a failed batch statement which is not whitelisted.

Reply:
Known issue: Ericsson/ecaudit#11